packages/ui-a11y-content/src/ScreenReaderContent/__tests__/ScreenReaderContent.test.tsx

```diff
@@ -28,7 +28,6 @@ import { ScreenReaderContent } from '../index'
 
 describe('<ScreenReaderContent />', async () => {
   it('should render the specified tag when `as` prop is set', async () => {
-    // @ts-expect-error ts-migrate(2554) FIXME: Expected 2 arguments, but got 1.
     const subject = await mount(<ScreenReaderContent as="div" />)
     expect(subject.getDOMNode()).to.have.tagName('div')
   })
```

Comment:
When I remove the `ts-expect-error on line 36 in this file I get this error: `  Property 'hidden' does not exist on type` ...

Reply:
Thanks for pointing it out, forgot to add the OtherHTMLAttributes to the components. Fixed!